benchmarks/synth-bm/Cargo.toml

```diff
@@ -17,6 +17,10 @@ near-jsonrpc-client = "0.14.0"
 near-jsonrpc-primitives = "0.27"
 near-primitives = "0.27"
 rand = "0.8.4"
+reqwest = "0.12"
 tokio = { version = "1.40.0", features = ["full"] }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
+
+[dev-dependencies]
+indoc = "2"
```

Comment:
It is somewhat unfortunate that `synth-bm` is not part of the overall workspace. Any reason why you chose to have this crate be entirely separate?

Reply:
IIRC it was the quickest way to get started and then the question of including er excluding it didn't get enough attention. I can include it once the outstanding PRs are merged #12887.